checklist.md

```diff
@@ -23,7 +23,7 @@ public class Main {
 
 #### Write informative messages in methods.
 Use english only and make them informative:
-message should indicate what type of `Machine` is working right now `Truck`, `Bulldozer` or `Excavator`.
+message should indicate what type of `core.basesyntax.Machine` is working right now `Truck`, `Bulldozer` or `Excavator`.
```

Comment:
Don't change checklist.md. Revert changes back

Reply:
not fixed. To revert changes copy the entire contents of the file from the branch's master file to your branch file